core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -773,28 +827,128 @@ export class LoggedInUser extends BaseUser {
   /**
    * Navigates to the Forum page using the oppia website footer.
    */
-  async navigateToForumPageViaFooter(): Promise<void> {
+  async clickOnForumLinkInFooter(): Promise<void> {
     await Promise.all([
       this.page.waitForNavigation(),
       await this.clickOn(footerForumlink),
     ]);
 
-    expect(this.page.url()).toBe('https://groups.google.com/g/oppia');
+    expect(this.page.url()).toBe(googleGroupsOppiaUrl);
   }
 
   /**
    * Navigates to the GetStarted page using the oppia website footer.
    */
-  async navigateToGetStartedPageViaFooter(): Promise<void> {
+  async clickOnGetStartedLinkInFooter(): Promise<void> {
     await this.page.waitForSelector(footerGetStartedLink);
     await this.clickButtonToNavigateToNewPage(
       footerGetStartedLink,
-      'About Oppia button in the About Menu on navbar',
+      'Get Started link in the footer',
       getStartedUrl,
       'Get Started'
     );
   }
 
+  /**
+   * Navigates to the GetStarted page using the oppia website footer.
+   */
+  async navigateToCreatorGuidelinesPageViaFooter(): Promise<void> {
+    await this.page.waitForSelector(footerCreatorGuidelinesLink);
+    await this.clickButtonToNavigateToNewPage(
+      footerCreatorGuidelinesLink,
+      'Creator Guidelines link in the footer',
+      creatorGuidelinesUrl,
+      'Creator Guidelines'
+    );
+  }
+
+  /**
+   * Navigates to the Teach page using the oppia website footer.
+   */
+  async clickOnForParentsSlashTeachersLinkInFooter(): Promise<void> {
+    await this.page.waitForSelector(footerCreatorGuidelinesLink);
+    await this.clickButtonToNavigateToNewPage(
+      footerTeachLink,
+      'For Parents/Teachers link in footer',
+      teachUrl,
+      'Oppia for Parents, Teachers, and Guardians'
+    );
+  }
+
+  /**
+   * Navigates to the Terms page using the oppia website footer.
+   */
+  async navigateToTermsPageViaFooter(): Promise<void> {
+    await this.page.waitForSelector(footerCreatorGuidelinesLink);
+    await this.clickButtonToNavigateToNewPage(
+      footerTermsLink,
+      'Terms of use link in footer',
+      termsUrl,
+      'Terms of Use'
+    );
+  }
+
+  /**
+   * Navigates to the Privacy Policy page using the oppia website footer.
+   */
+  async navigateToPrivacyPolicyPageViaFooter(): Promise<void> {
```

Comment:
This function has never been called as well as the function above for the Terms Page. Call them in the file click-more-links-in-oppia-footer.spec.ts. Also, change the functions' names to "clickOn..." so we know it tests the functionality of the button.

Reply:
Done